extensions/todoist/src/create-task.tsx

```diff
@@ -146,8 +147,13 @@ export default function CreateTask() {
       />
 
       <Form.Dropdown id="priority" title="Priority" value={priority} onChange={setPriority}>
-        {priorities.map(({ value, name }) => (
-          <Form.Dropdown.Item value={String(value)} title={name} key={value} />
+        {priorities.map(({ value, name, color, icon }) => (
+          <Form.Dropdown.Item
+            value={String(value)}
+            title={name}
+            key={value}
+            icon={color && { source: icon ? icon : Icon.Dot, tintColor: color }}
```

Comment:
Since all priorities have an `icon` property, is it useful to have the ternary condition here?